iotkit/helloworld/display/display.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Package display is a driver for Grove LCD RGB backlight display.
+//
+// More information about the display is at http://wiki.seeed.cc/Grove-LCD_RGB_Backlight/.
+package display
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/exp/io/i2c"
+	"golang.org/x/exp/io/i2c/driver"
+)
+
+const (
+	returnHome     = 0x02
+	entryModeSet   = 0x04
+	displayControl = 0x08
+
+	functionSet = 0x20
+
+	entryRight          = 0x00
+	entryLeft           = 0x02
+	entryShiftIncrement = 0x01
+	entryShiftDecrement = 0x00
+
+	displayOn  = 0x04
+	displayOff = 0x00
+	cursorOn   = 0x02
+	cursorOff  = 0x00
+	blinkOn    = 0x01
+	blinkOff   = 0x00
+
+	displayMove = 0x08
+	cursorMove  = 0x00
+	moveRight   = 0x04
+	moveLeft    = 0x00
+
+	mode8bit = 0x10
+	mode4bit = 0x00
+
+	line2 = 0x08
+	line1 = 0x00
+
+	dots10 = 0x04
+	dots8  = 0x00
+
+	// addresses
+	lcdAddr = 0x3e
+	rgbAddr = 0x62
```

Comment:
Are all/some of these constants something that should live in an `cloud/iokit` package?